src/app/common/duck/actions.ts

```diff
@@ -0,0 +1,9 @@
+import { createActions } from "reduxsauce";
+
+const { Creators, Types } = createActions({
+  alertSuccess: ["value"],
+  alertError: ["value"],
+  alertClear: ["value"]
```

Comment:
If I'm thinking correctly, we might want to change "value" to something else, like "success", "error" etc

Reply:
Yes value should be changed here.